src/Sulu/Bundle/AdminBundle/Resources/js/components/CharacterCounter/tests/__snapshots__/CharacterCounter.test.js.snap

```diff
@@ -1,41 +1,61 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Should show a negative count 1`] = `
-<label
-  class="characterCounter exceeded"
->
-  -8 sulu_admin.characters_left
-</label>
+<div>
+  <label
+    class="characterCounter exceeded"
+  >
+    -8
+     
+    sulu_admin.characters_left
```

Comment:
@b3h3m0th Do we know why it adds 2 line breaks here instead of just the space? Can not understand it as there seems no breaks in the file itself: https://github.com/sulu/sulu/blob/544a6714d9ea4500e7cfdd5913d31c0193511858/src/Sulu/Bundle/AdminBundle/Resources/js/components/CharacterCounter/CharacterCounter.js#L26

Is this a default react behaviour?

Reply:
That's weird, I don't know either. It's definitely not default react behaviour as you can see in [this sandbox](https://codesandbox.io/s/youthful-shtern-idsxis). Is it possible that RTL does something odd there?

Reply:
Found this issue: https://github.com/testing-library/react-testing-library/issues/53 but the are not targeting how the snapshot is created they seems just try to fix that the selectors still work there.

Maybe we can workaround by using: `{charactersLeft + ' ' + translate('sulu_admin.characters_left')}` in the component currently.

Reply:
Yes that fixed it. I updated the component and it's shapshot in [the following commit](https://github.com/sulu/sulu/pull/6730/commits/8c44a4678cdeb2ebe515209580dc28e2ba63370d).